docs/FAQ.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 3
 
 ### What is OpenCost?
 
-OpenCost is a vendor-neutral open source project for measuring and allocating infrastructure and container costs in Kubernetes environments.
+OpenCost is a vendor-neutral open source project for measuring and allocating infrastructure, container and cloud costs in Kubernetes environments.
```

Comment:
Thoughts on this?

"OpenCost is a vendor-neutral open source project for measuring and allocating Kubernetes and cloud infrastructure costs."

Reply:
Updated accordingly.